README.md

```diff
@@ -118,6 +134,24 @@ LINK NODE1-NODE2
     NODES NODE1 NODE2
     TARGET prometheus:http:localhost:9090:192.168.178.32:Gi0/1:ifHCInOctets
 ```
+
+This example uases a free-text query to get the throughput of the busiest link in a LAG as a dual query, getting both In and Out:
+```
+LINK NODE1-NODE2
+    NODES NODE1 NODE2
+    TARGET prometheus:http:localhost:9090:max(irate(ifHCOutOctets{ifAlias=~".*ae232.*",instance="192.168.178.32"}[10m]))*8:max(irate(ifHCInOctets{ifAlias=~".*ae232.*",instance="192.168.178.32"}[10m]))*8
+```
+In this case we're identifying the interfaces that make up this LAG using a regex query on the ifAlias label.
+
+Modifed to use single query that gets the maximium of both In & Out throughput:
+```
+LINK NODE1-NODE2
+    NODES NODE1 NODE2
+    TARGET max(irate(ifHCOutOctets{ifAlias=~".*ae232.*",instance="192.168.178.32"}[10m]) or irate(ifHCOutOctets{ifAlias=~".*ae232.*",instance="192.168.178.32"}[10m]))*8
```

Comment:
I think that it's missing at the beginning the whole `prometheus:http:localhost:9090:` part to be
```
TARGET prometheus:http:localhost:9090:max(irate(ifHCOutOctets{ifAlias=~".*ae232.*",instance="192.168.178.32"}[10m]) or irate(ifHCOutOctets{ifAlias=~".*ae232.*",instance="192.168.178.32"}[10m]))*8
```

Reply:
Fixed